common/jvm/src/test/scala/com/redi/scalademo/business/NumericStringValidatorTest.scala

```diff
@@ -0,0 +1,38 @@
+package com.redi.scalademo.business
+
+import org.scalatest.{Matchers, path}
+
+class NumericStringValidatorTest extends path.FunSpec with Matchers {
```

Comment:
There are some features of scala that aren't supported in scalaJS so I think I agree with Shogo to put it in Shared and let it run on both platforms.